openshift-container-platform-4/policies/SC-Systems_and_Communications_Protection/component.yaml

```diff
@@ -278,39 +278,33 @@
   narrative:
     - key: a
       text: |
-        This control is inherited from the network and IaaS provider.
-        To assist with this organizational control, documentation is being
-        planned. Progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-425
+        Implementing a managed interface for each external telecommunication service
+        is addressed at the network infrastructure layer and is an organizational
+        control outside the scope of OpenShift configuration.
     - key: b
       text: |
-        This control is inherited from the network and IaaS provider.
-        To assist with this organizational control, documentation is being
-        planned. Progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-425
+        Establishing a traffic flow policy for each managed interface for each
+        external telecommuncation service is an organizational control outside the
+        scope of OpenShift configuration and is typically handled at the network
+        infrastructure layer.
     - key: c
       text: |
-        This control is inherited from the network and IaaS provider.
-        To assist with this organizational control, documentation is being
-        planned. Progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-425
+        Protecting the confidentiality and integrity of the information being transmitted
+        across each interface is an organizational control outside the
+        scope of OpenShift configuration and is typically handled at the network
+        infrastructure layer.
```

Comment:
Could we also mention here that for the control plane components in OpenShift we do use mutual TLS? Or is that not in scope in this doc?

Reply:
@JAORMX this control is about external telecommunication service(s). Per our conversation, OCP shouldn't or does not interface with external telecommunication service(s) which is typically done at the switch or router layer. Do you disagree?

Reply:
Oh, you're right. I got confused there. Then this is good.